src/test/java/org/llorllale/cactoos/matchers/HasLinesTest.java

```diff
@@ -50,10 +50,10 @@
      */
```

Comment:
Do we still need 'PMD.AvoidDuplicateLiterals' here?

Reply:
@ilyakharlamov removed